lib/kafo/configuration.rb

```diff
@@ -88,6 +88,10 @@ def modules
       @modules ||= @data.keys.map { |mod| PuppetModule.new(mod, PuppetModule.find_parser, self).parse }.sort
     end
 
+    def module(name)
+      modules.find { |m| m.name == name }
```

Comment:
Updated, now #param uses this method to find the module, then calls params on it.